src/GitLink/Extensions/RepositoryExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace GitLink
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Catel;
+    using Catel.IO;
+    using LibGit2Sharp;
+
+    internal static class RepositoryExtensions
+    {
+        internal static string GetRepoNormalizedPath(this Repository repository, string path)
+        {
+            Argument.IsNotNull(nameof(repository), repository);
```

Comment:
You can use Argument.IsNotNull(() => repository) as well. It will be weaved by fody to what you've written (but it's a matter of preference).